packages/liveblocks-react-ui/src/components/Composer.tsx

```diff
@@ -325,9 +325,7 @@ const ComposerWithContext = forwardRef<
     }, []);
 
     const handleEditorClick = useCallback(
-      (event: MouseEvent<HTMLDivElement>) => {
-        event.stopPropagation();
```

Comment:
Did you see any issues after this change?

Reply:
Nothing strange yet. But, will revert this change since we won't be focusing the composer on click.

Reply:
Nothing strange yet. I could not think of how this would affect other things but I am ready to be surprised 😬

Reply:
Since we're hiding the composers by default now, this change is not needed anymore, should we keep it as it was before until we need it?

Reply:
We still want to select the associated mark node (so that the selected thread is active), so this will still be needed. Otherwise, clicking on the composer will not result in the thread being active.